wgpu-core/src/command/transfer.rs

```diff
@@ -594,6 +597,26 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
         if destination_offset % wgt::COPY_BUFFER_ALIGNMENT != 0 {
             return Err(TransferError::UnalignedBufferOffset(destination_offset).into());
         }
+        if !device
+            .downlevel
+            .flags
+            .contains(wgt::DownlevelFlags::UNRESTRICTED_INDEX_BUFFER)
+            && (src_buffer.usage.contains(BufferUsages::INDEX)
```

Comment:
Mixed imported and fully qualified usage of `BufferUsages` - prefer `wgt::BufferUsages`